modules/end-user-guide/partials/proc_using-a-web-reference-for-your-ide.adoc

```diff
@@ -0,0 +1,16 @@
+[id="using-a-web-reference-for-your-ide_{context}"]
+=== Using a web reference for your IDE
+
+Rather than pointing to a plug-in in your plug-in registry, you can point at a YAML file directly with the `reference` directive.
```

Comment:
Would it help users to know _why_ they might want to do this, or is it reasonable to expect that they already know why?